recommendations.md

```diff
@@ -23,3 +23,40 @@ Referencing by tag (`:latest` or `:v1.2.3`) means that an owner of that image
 can push a new image to that tag, and all Tasks that reference the image by
 that tag will start using it immediately. This can lead to unexpected Task
 failures, or silent behavior changes, including security-sensitive changes.
+
+## Run as non root and non privileged
+
+One of the security best practices of containers is to run them as a
+non-root user. Usually this is achieved by having a user defined in
+your image and having it referred in your image configuration. You can
+see
+[here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user)
+for details on best practice with `Dockerfile`s.
+
+You should also avoid as much as possible to run containers as
```

Comment:
Maybe you want to add a note about the `builders` task which in most case would need to be run as priv/root?